testclient/mainwindow.ui

```diff
@@ -137,6 +137,13 @@
              </widget>
             </item>
             <item>
+             <widget class="QLabel" name="bedTempLB">
```

Comment:
These Gui changes are not connected to anything can we do this on a commit that also contains logic to make them function.